kafka/producer.go

```diff
@@ -360,6 +360,24 @@ func (p *Producer) Close() {
 	C.rd_kafka_destroy(p.handle.rk)
 }
 
+const (
+	// RdKafkaPurgeFQueue - Purge from queue
+	RdKafkaPurgeFQueue = C.RD_KAFKA_PURGE_F_QUEUE
+	// RdKafkaPurgeFInflight - Purge from inflight
+	RdKafkaPurgeFInflight = C.RD_KAFKA_PURGE_F_INFLIGHT
+)
+
+// Purge messages from queue.
```

Comment:
Use this doc string:
```
// Purge messages currently handled by this producer instance.
//
// flags is a combination of PurgeQueue, PurgeInFlight and PurgeNonBlocking.
//
// The application will need to call Poll(), Flush() or read the Events() channel
// after this call to serve delivery reports for the purged messages.
//
// Messages purged from internal queues fail with the delivery report
// error code set to ErrPurgeQueue, while purged messages that
// are in-flight to or from the broker will fail with the error code set to
// ErrPurgeInflight.
//
// Warning: Purging messages that are in-flight to or from the broker
//          will ignore any sub-sequent acknowledgement for these messages
//          received from the broker, effectively making it impossible
//          for the application to know if the messages were successfully
//          produced or not. This may result in duplicate messages if the
//          application retries these messages at a later time.
//
// Note: This call may block for a short time while background thread
//       queues are purged.
//
// Returns nil on success, ErrInvalidArg if the purge flags are invalid or unknown.
```